### ISSUE TITLE
Fails to import settings file

### ISSUE THREAD

**Issue by Azirine** [sereneblue/chameleon#521]
## Expected Behavior
Downloading settings file from [here](https://sereneblue.github.io/chameleon/wiki/getting-started) and importing into Chameleon will import settings into Chameleon.

## Current Behavior
Settings cannot be imported into Chameleon. Gets error message, 'Could not import file'.

**Comment by sereneblue** [comment]
Hi @Azirine,

I've tried importing both settings with v0.22.12 and could not reproduce this issue. What version of Chameleon are you using?

**Comment by sereneblue** [comment]
@Azirine Thanks for the detailed information. Chameleon does support importing settings from older versions (even pre-v0.20). I've tried importing the settings using Firefox on Windows and I encountered successful/unsuccessful imports with the same file. After a few imports, it seems the export is also missing the version information so there seems to be a bug somewhere that needs to be resolved. In the meantime, I'll update the settings on the wiki.
